geode-redis/src/main/java/org/apache/geode/redis/internal/RedisCommandType.java

```diff
@@ -142,6 +142,7 @@
   RENAME(new RenameExecutor(), SUPPORTED, new ExactParameterRequirements(3)),
   TTL(new TTLExecutor(), SUPPORTED, new ExactParameterRequirements(2)),
   TYPE(new TypeExecutor(), SUPPORTED, new ExactParameterRequirements(2)),
+  UNLINK(new DelExecutor(), SUPPORTED, new MinimumParameterRequirements(2)),
```

Comment:
We said it was `Unsupported` in the README, but added it as `Supported` here.  What did we ultimately decide?

Reply:
Oops - good catch. Thanks